soc/riscv/riscv-privilege/hpmicro/Kconfig.soc

```diff
@@ -0,0 +1,62 @@
+# Copyright 2022 hpmicro
+# SPDX-License-Identifier: Apache-2.0
+
+choice
+prompt "HPMicro HPM6700_6400 SoC Selection"
+depends on SOC_SERIES_HPMICRO
+
+config SOC_HPM6750
+	bool "HPMicro HPM6750 SoC implementation"
+	select ATOMIC_OPERATIONS_BUILTIN
+	select INCLUDE_RESET_VECTOR
+	select RISCV_ISA_EXT_M
+	select RISCV_ISA_EXT_A
+	select RISCV_ISA_RV32I
+	select HAS_HPMSDK_UART
+	select HAS_HPMSDK_GPIO
+	select HAS_HPMSDK_PINCTRL
+	select HAS_HPMSDK_PDM
+	select HAS_HPMSDK_PLLCTL
+	select HAS_HPMSDK_PMP
```

Comment:
drivers need to select what they need, not SoC.

Reply:
ping, these options should not be here but in drivers.

Reply:
Okay, I'll remove the driver configuration, but I need to keep the soc configuration.